docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-dagster.md

```diff
@@ -38,120 +38,221 @@ by signing up for the trial.
+
+       ![image](https://github.com/user-attachments/assets/97b74b86-df94-47e5-8ae2-de7cc47f56d8)
+
+1. Run the pipeline.
+
+   Now that you have a running instance of Dagster, we can run our data pipeline.
+
+   To run the pipeline, go to **Assets** and click the **Materialize** button in the top right. In Dagster, materialization refers to executing the code associated with an asset to produce an output.
+
+   ![image](https://github.com/user-attachments/assets/79416fb7-8362-4640-b205-e59aa7ac785c)
+
+   You will see the following logs in your command line:
+
+   ![image](https://github.com/user-attachments/assets/f0e3bec8-f702-46a6-b69f-194a1dacf625)
+
+   Want to see real-world examples of dlt in production? Check out how dlt is used internally at Dagster in the [Dagster Open Platform](https://github.com/dagster-io/dagster-open-platform) project.
+
+
+:::info
+For a complete picture of Dagster's integration with dlt, please refer to their [documentation](https://docs.dagster.io/integrations/embedded-elt/dlt). This documentation offers a detailed overview and steps for ingesting GitHub data and storing it in Snowflake. You can use a similar approach to build your pipelines.
 :::
 
+### Frequently Asked Questions
+- **Can I remove the generated .dlt folder?**
+
+  Yes. Since dlt is compatible with .env files, you can use this for secrets required by both dagster and dlt.
+  
+- **I'm working with several sources – how can I best group these assets?**
+
+  To effectively group assets in Dagster when working with multiple sources, use the group_name parameter in your `@dlt_assets` decorator. This helps organize and visualize assets related to a particular source or theme in the Dagster UI. Here’s a simplified example:
+  
+  ```py
+  import dlt
+  from dagster_embedded_elt.dlt import dlt_assets
+  from dlt_sources.google_analytics import google_analytics
+  
+  # Define assets for the first Google Analytics source
+  @dlt_assets(
+      dlt_source=google_analytics(...),
+      dlt_pipeline=dlt.pipeline(...),
+      group_name='Google_Analytics'
+  )
+  def google_analytics_assets_1(context, dlt):
+      yield from dlt.run(context=context)
+  
+  # Define assets for the second Google Analytics source
+  @dlt_assets(
+      dlt_source=google_analytics(...),
+      dlt_pipeline=dlt.pipeline(...),
+      group_name='Google_Analytics'
+  )
+  def google_analytics_assets_2(context, dlt):
+      yield from dlt.run(context=context)
+  ```
+
+ 
+  
+- **How can I use bigquery_adapter with @dlt_assets in Dagster for partitioned tables?**
+   
+  To use `bigquery_adapter` with `@dlt_assets` in Dagster for partitioned tables, modify your resource setup to include `bigquery_adapter` with the partition parameter. Here's a quick example:  
+  
+  ```py
+  from dlt import bigquery_adapter
+  
+  # Configure the resource with partitioning
+  resource = bigquery_adapter(
+      dlt.resource(...),
+      partition="date"  # Specify partitioning by date
+  )
```

Comment:
extend this example, it's not very clear how to use this adapter and dagster assets, we don't show assets in this example